Bio/Blast/NCBIWWW.py

```diff
@@ -76,12 +82,16 @@ def qblast(program, database, sequence, url_base=NCBI_BLAST_URL,
     import time
 
     assert program in ['blastn', 'blastp', 'blastx', 'tblastn', 'tblastx']
+    if url_base == "https://blast.ncbi.nlm.nih.gov/Blast.cgi":
+        assert num_threads is None
```

Comment:
I think you could give an explicit error here, e.g.

```python
if url_base == "https://blast.ncbi.nlm.nih.gov/Blast.cgi" and num_threads is not None:
    raise ValueError("The num_threads option cannot be used on the NCBI servers")
```

Reply:
Sure, it is way better.